src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -13,10 +17,21 @@ import {AppConfigService} from '../config/configuration.services';
 export class AppConfig extends AbstractAppConfig {
 
   protected config: CaseEditorConfig;
+  public workallocationUrl: string;
 
-  constructor(private appConfigService: AppConfigService) {
+  constructor(
+    private appConfigService: AppConfigService,
```

Comment:
Can you also make this one `readonly`? 😃

Reply:
yes done